.gitmodules

```diff
@@ -10,3 +10,6 @@
 [submodule "coq-waterproof"]
 	path = coq-waterproof
 	url = git@github.com:impermeable/coq-waterproof.git
+[submodule "waterproof-exercises"]
+	path = waterproof-exercises
+	url = https://gitlab.tue.nl/impermeable/waterproof-exercises
```

Comment:
```suggestion
	url = git@gitlab.tue.nl:impermeable/waterproof-exercises.git
```

Reply:
It hung for me when using https (likely because it's prompting for password and that didn't work through `git submodule update`